### ISSUE TITLE
Generic template - accessibility - line length to wide

### ISSUE THREAD

**Issue by Dan-KeduTesting** [Buckinghamshire-Digital-Service/hr-service-desk-alpha#129]
accessibility guidelies says it should be less then 80 char per line

can line width be 650 px on desktop width please?

![Generic_template_-_line_lenght_to_wide](https://user-images.githubusercontent.com/2722522/65632585-ec78a800-dfd1-11e9-8c02-b365074d54eb.png)

**Comment by enginehouse** [comment]
Hi @Dan-KeduTesting  - I haven'thad a chance to style this section correctly yet - I would suggest holding off for a while

**Comment by enginehouse** [comment]
GIven a max-width on body content wrapper

**Comment by Dan-KeduTesting** [comment]
this is still an accessibility issue for body text. guidelines says that a line should have max of 80 char

font size for body text looks a little small... can we increase font size for the body text

**Comment by Dan-KeduTesting** [comment]
failed testing

line width is over 80 char per line. 

To fix either change width to 600px or suggest increasing font size by 1px (font seems a little to small) and reducing with slightly (by 50px maybe)

also at viewports around 780px there is hardly any margin left and right  of content so suggest increasing margin here
